### PR TITLE
Rename gogs.go to main.go with all references

### PR DESCRIPTION
**Pull request by metalmatze**
Just to stick to the Go rule of thumb for the `main.go`.

### CODE REVIEW

**Comment by xinity** [comment]
LGTM

File: scripts/build.sh [hunk 1]
```diff
@@ -1,9 +1,10 @@
+#!/usr/bin/env bash
 outPath=./output
 
 rm -rf $outPath
 mkdir $outPath
 
-go build ../gogs.go
+go build ../main.go
```
Comment (andreynering) [inline scripts/build.sh]: Shouldn't this have `-o gitea.go`?

File: scripts/build.sh [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 PLATFORM=`uname | cut -d _ -f 1`
 if [ $PLATFORM = "MINGW32" ] || [ $PLATFORM = "MINGW64" ] || [ $PLATFORM = "CYGWIN" ]; then
 	GOGS_EXE=gogs.exe
```
Comment (andreynering) [inline scripts/build.sh]: I think we should review what this does, too.

File: scripts/build_freebsd.sh [hunk 3]
```diff
@@ -1,11 +1,12 @@
+#!/usr/bin/env bash
 outPlattform=freebsd
 outArch=amd64
 outPath=./output_$outPlattform_$outArch
 
 rm -rf $outPath
 mkdir $outPath
 
-CGO_ENABLED=0 GOOS=$outPlattform GOARCH=$outArch go build ../gogs.go
+CGO_ENABLED=0 GOOS=$outPlattform GOARCH=$outArch go build ../main.go
```
Comment (andreynering) [inline scripts/build_freebsd.sh]: Same here

File: scripts/build_linux64.sh [hunk 4]
```diff
@@ -1,11 +1,12 @@
+#!/usr/bin/env bash
 outPlattform=linux
 outArch=amd64
 outPath=./output_$outPlattform_$outArch
 
 rm -rf $outPath
 mkdir $outPath
 
-CGO_ENABLED=0 GOOS=$outPlattform GOARCH=$outArch go build ../gogs.go
+CGO_ENABLED=0 GOOS=$outPlattform GOARCH=$outArch go build ../main.go
```
Comment (andreynering) [inline scripts/build_linux64.sh]: Same here

**Comment by metalmatze** [comment]
I just updated the `scripts/` because it had a reference there.
We really should get rid of the `scripts/` and try to do this with a good `Makefile`.

File: scripts/build.sh (hunk 1 above)
Comment (tboerger) [reply scripts/build.sh]: AFAIK it should take the directory name, but you can prove that I'm wrong :)

File: scripts/build.sh (hunk 2 above)
Comment (tboerger) [reply scripts/build.sh]: This also needs to be fixed.

**Review by tboerger — CHANGES_REQUESTED** [review]
Please replace the remaining `gogs` references within the scripts folder.

**Comment by tboerger** [comment]
Beside the pending gogs references in scripts it needs a rebase.

File: scripts/build.sh (hunk 1 above)
Comment (bkcsoft) [reply scripts/build.sh]: Even though the file is named `main.go` this is still incorrect, since @tboerger is correct :slightly_smiling_face:

**Comment by bkcsoft** [comment]
Makefile is also outdated :slightly_smiling_face:

**Comment by metalmatze** [comment]
Updated my branch with a force push. Rebased against current master and improved all the things said above! 👍

**Comment by strk** [comment]
More references missing, "make" ends with: `cp '/home/strk/.go/bin/gogs'`
Check the Makefile please

**Comment by metalmatze** [comment]
You're right. But I don't want to touch anything in `scripts/` or the `Makefile`. @tboerger is already working on replacing the existing `Makefile` completely. @tboerger could just merge this and then add his new `Makefile`. Don't want to waste energy there. :)

**Comment by tboerger** [comment]
Yes, don't touch the makefile, merge this pr :)

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
LGTM will integrate the makefile and build scripts stuff into my PR.

**Comment by strk** [comment]
LGTM

**Comment by DblK** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

**Pull request merged**